release-notes.md

```diff
@@ -1,5 +1,16 @@
 # Release notes
 
+## June 17, 2019
+
+The following updates were made in this release:
+
+**iOS Places 1.1.0**
+
+* Added a new API to return an error code when there is a failure retrieving nearby places.
+* When privacy status changes to opt-out, all Places-related data will now be wiped from the device.
+* Fixed a bug sometimes causing Places events following a first launch to be lost due to bad network conditions.
```

Comment:
Please reword to read as follows:

"Fixed an issue that, after a first launch, sometimes caused Places events to be lost due to bad network conditions."